test/libsolidity/SolidityOptimizer.cpp

```diff
@@ -54,13 +54,13 @@ class OptimizerTestFramework: public SolidityExecutionFramework
 		unsigned const _optimizeRuns = 200
 	)
 	{
-		bool const c_optimize = m_optimize;
-		unsigned const c_optimizeRuns = m_optimizeRuns;
-		m_optimize = _optimize;
-		m_optimizeRuns = _optimizeRuns;
+		OptimiserSettings previousSettings = std::move(m_optimiserSettings);
+		// This uses "none" / "full" while most other test frameworks use
```

Comment:
This comment should be changed with https://github.com/ethereum/solidity/pull/6334.